panel/widgets/chat.py

```diff
@@ -0,0 +1,1646 @@
+"""The chat module provides components for building and using chat interfaces
```

Comment:
When I run the automated tests to generate videos of `chat_memory.py` I can see that the `ChatFeed` does not scroll to the end. I.e. the user ends up not being able to see what is written.

[6facf4c1fa752e60f478ebf366720e4d.webm](https://github.com/holoviz/panel/assets/42288570/9ab4dc7b-6b41-404a-815a-41f5bd7b9402)

![test-finished-1](https://github.com/holoviz/panel/assets/42288570/b29ebd3f-558b-46ce-8083-181cb6c9c5e0)


```python
"""
Demonstrates how to use the ChatInterface widget to create a chatbot using
OpenAI's GPT-3 API with LangChain.
"""

import panel as pn
from langchain.chains import ConversationChain
from langchain.chat_models import ChatOpenAI
from langchain.memory import ConversationBufferMemory

pn.extension(design="material")


async def callback(contents: str, user: str, instance: pn.widgets.ChatInterface):
    await chain.apredict(input=contents)


chat_interface = pn.widgets.ChatInterface(callback=callback, callback_user="ChatGPT")
chat_interface.send(
    "Send a message to get a reply from ChatGPT!", user="System", respond=False
)

callback_handler = pn.widgets.langchain.PanelCallbackHandler(
    chat_interface=chat_interface
)
llm = ChatOpenAI(streaming=True, callbacks=[callback_handler])
memory = ConversationBufferMemory()
chain = ConversationChain(llm=llm, memory=memory)
chat_interface.servable()
```

Reply:
Maybe try tweaking `auto_scroll_limit`?